core/nodepool/config/templates/stack-template.json

```diff
@@ -361,6 +368,7 @@
                   "Action": [
                     "autoscaling:DescribeAutoScalingGroups",
```

Comment:
@redbaron Thanks, good question 👍 
The bigger a k8s cluster is, the more CA's resource usage is.
So, I guess one wants a dedicated node pool with a single worker node which can be recreated easier than controller nodes and used to run CA which is possibly scaled-up/down by a vertical-pod-autoscaler.
That's why I tried to support running CA in worker nodes, even though it isn't the default setup.
What do you think? 😃